source/MaterialXView/Glsl/GLUtil/SimpleWindow_windows.cpp

```diff
@@ -0,0 +1,115 @@
+#include <MaterialXView/Glsl/GLUtil/HardwarePlatform.h>
+
+#if defined(OSWin_)
+
+#include <Windows.h>
+#include <MaterialXView/Glsl/GLUtil/SimpleWindow.h>
+#include <MaterialXView/Glsl/GLUtil/WindowWrapper.h>
+#include <string>
+
+namespace MaterialX
+{
+SimpleWindow::SimpleWindow()
+{
+    // Give a unique ID to this window.
+    //
+    static unsigned int windowCount = 1;
+    _id = windowCount;
+    windowCount++;
+
+    // Generate a unique string for our window class. 
+    sprintf_s(_windowClassName, "_SW_%lu", _id);
+}
+
+// No-op window procedure
+LRESULT CALLBACK NoOpProc(HWND hWnd, UINT msg, WPARAM wParam, LPARAM lParam)
```

Comment:
If we ever want to extend this, we can add in a more complex event handler. For now it does nothing.